src/components/Compare/CompareTable.tsx

```diff
@@ -232,41 +270,16 @@ const CompareTable = (props: {
           )}
         </HeaderContainer>
       )}
-      <LocationTable
-        firstColumnHeader={firstColumnHeader}
-        setSorter={setSorter}
-        setSortDescending={setSortDescending}
-        columns={columns}
-        isModal={props.isModal}
-        {...arrowContainerProps}
-        pinnedLocation={currentLocation}
-        sortedLocations={sortedLocations}
-        numLocations={locationsViewable}
-        stateName={props.stateName}
-        setSortByPopulation={setSortByPopulation}
-        sortByPopulation={sortByPopulation}
-        isHomepage={props.isHomepage}
-        geoScope={props.geoScope}
-        homepageScope={homepageScope}
-        region={region}
-      />
+      {!props.isModal && showViewMore ? (
+        <ExpandableContainer {...containerProps}>
+          <LocationTable {...locationTableProps} />
+        </ExpandableContainer>
+      ) : (
+        <LocationTable {...locationTableProps} />
+      )}
       {!props.isModal && (
         <Footer>
-          <div>
-            <FooterText>
-              Displaying <strong>{amountDisplayed}</strong> of{' '}
-              <strong>{sortedLocationsArr.length}</strong>{' '}
-            </FooterText>
-            {showViewMore && (
-              <TextButton
-                onClick={onClickShowAll}
-                trackingCategory={EventCategory.COMPARE}
-                trackingLabel="view all regions button"
-              >
-                {props.viewMoreCopy}
-              </TextButton>
-            )}
-          </div>
+          <div /> {/* Need empty div to right align share button. */}
```

Comment:
[can repurpose ShareBlock](https://github.com/covid-projections/covid-projections/blob/develop/src/components/Explore/Explore.style.ts#L26) (used in explore footer to align share button to the right when there's no legend) -- and maybe move it to somewhere more central (like the share button style file, or the footer styles, etc)